### ISSUE TITLE
encodeURI breaks aws presigned URLs as assets

### ISSUE THREAD

**Issue by oreporan** [inlife/nexrender#703]
**Describe the bug**
A clear and concise description of what the bug is, and ways to reproduce it.
Appears that [this](https://github.com/inlife/nexrender/pull/380) PR breaks assets that are AWS s3 presigned_urls

This is an example image in the presigned format:
```
https://mys3.s3.us-west-2.amazonaws.com/xx/yy/zz.mp4?AWSAccessKeyId=myaccesskey&Expires=1645572153&Signature=xx%2FrAv28PsXy%2BXU7rC2g%3D
```
The problem occurs around the signature which after the `encodeURI()` becomes: 

```
xx%252FrAv28PsXy%252BXU7rC2g%253D
```

**Information about environment**
* Which version of nexrender are you using?

```
  "@nexrender/action-cache": "^1.29.0",
        "@nexrender/action-copy": "^1.0.0",
        "@nexrender/action-encode": "^1.1.4",
        "@nexrender/action-upload": "^1.0.0",
        "@nexrender/core": "^1.33.0",
        "@nexrender/provider-s3": "^1.21.4",
```

* What kind of the setup do you have? (server+worker or cli) - only core
* Are you using pre-built binaries or globally installed npm packages? - npm
* Which kind (and version) of operating system you are running it in? - windows
* Any additional information that might be helpful

**Expected behavior**
maybe the option to skip encoding the URI?

**Comment by oreporan** [comment]
Could be related to https://github.com/inlife/nexrender/issues/687

**Comment by ivansenic** [comment]
Hit the same issue long ago. Can we completely delete the handling of the encoding URLs on the worker side. The current way how the discovery if the implementation is anyway wrong here in [download.js](https://github.com/inlife/nexrender/blob/aee34bab9e112d980611afcadc085d5c3ad7d440/packages/nexrender-core/src/tasks/download.js#L70). I opt in completely removing this. Links should be already encoded when being added to the job.

**Comment by oreporan** [comment]
I think this might have been fixed

**Comment by thenovelnomad** [comment]
this is fixed in 1.46.3.
